Fluentify.Web/Pages/Login.cshtml

```diff
@@ -0,0 +1,108 @@
+﻿@page
+@model Fluentify.Web.Pages.LoginUserModel
+@{
+ViewData["Title"] = "Sign In";
+}
+
+<style type="text/css">
+    body {
+        background-color: #0A3D62;
+        color: #F5F5F5;
+    }
+    .error {
+        color: red;
+        font-size: 10px;
+    }
+
+    .form-group {
+        margin-top: 20px;
+    }
+
+    .SubmitButton {
+        position: relative;
+        z-index: 1;
+        display: inline-block;
+        background-color: #007bff;
+        color: #fff;
+        text-align: center;
+        text-transform: uppercase;
+        letter-spacing: 0.15em;
+        font-weight: 600;
+        padding: 1em 2em;
+        margin: 0 auto;
+        overflow: hidden;
+        border-radius: 0.5em;
+        transition: all 0.3s ease-in-out;
+        -webkit-box-shadow: 0px 5px 40px -10px rgba(0, 123, 255, 0.4);
+        box-shadow: 0px 5px 40px -10px rgba(0, 123, 255, 0.4);
+        border: none;
+    }
+
+    .SubmitButton:hover {
+        background-color: #fff;
+        color: #007bff;
+        -webkit-box-shadow: none;
+        box-shadow: none;
+    }
+
+    .SubmitButton span.btn-text {
+        display: block;
+    }
+
+    .SubmitButton span.btn-loader {
+        position: absolute;
+        top: 50%;
+        left: 50%;
+        transform: translate(-50%, -50%);
+        display: none;
+    }
+
+    .SubmitButton.loading span.btn-text {
+        opacity: 0;
+        visibility: hidden;
+    }
+
+    .SubmitButton.loading span.btn-loader {
+        display: block;
+    }
+</style>
```

Comment:
It will be better to move this `CSS` classes to one file, for example `site.css` or smth like that.